src/kakarot/evm.cairo

```diff
@@ -278,3 +287,28 @@ namespace EVM {
         );
     }
 }
+
+namespace Internals {
+    func is_valid_jumpdest{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr,
+        valid_jumpdests: DictAccess*,
+    }(code_address: felt, is_create: felt, index: felt) -> felt {
+        alloc_locals;
+        let (is_cached) = dict_read{dict_ptr=valid_jumpdests}(index);
```

Comment:
so this is more `is_valid` like below

Reply:
not really, because it could be valid, but not cached. here the semantic of the variable name really carries the intent.